src/components/edit-details/index.js

```diff
@@ -15,6 +15,9 @@ import {
 import './style.css';
 export default class EditDetails extends Component {
   onSearchClick () {
+    dataLayer.push({
+      event: 'travelInfoUpdate'
```

Comment:
@Kumjami do you think we could send the passenger mix date info to dataLayer?
(*does not have to be done now... but what are your thoughts?*)